src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/homeoffice/HomeOfficeReinstateAppealPersonalisationTest.java

```diff
@@ -113,17 +113,19 @@ public void should_return_given_email_address() {
         List<State> apcEmail = newArrayList(
             State.APPEAL_SUBMITTED,
             State.AWAITING_RESPONDENT_EVIDENCE,
-            State.PENDING_PAYMENT
-
-
+            State.PENDING_PAYMENT,
+            State.CLARIFYING_QUESTIONS_ANSWERS_SUBMITTED,
+            State.AWAITING_CLARIFYING_QUESTIONS_ANSWERS
         );
 
         List<State> lartEmail = newArrayList(
             State.CASE_BUILDING,
             State.CASE_UNDER_REVIEW,
             State.RESPONDENT_REVIEW,
             State.LISTING,
-            State.SUBMIT_HEARING_REQUIREMENTS
+            State.SUBMIT_HEARING_REQUIREMENTS,
+            State.REASONS_FOR_APPEAL_SUBMITTED,
+            State.LISTING
```

Comment:
This is a duplicate of line 125. Should it be 'State.REASONS_FOR_APPEAL_SUBMITTED' ?